packages/nx/src/tasks-runner/life-cycles/view-logs-utils.ts

```diff
@@ -5,7 +5,8 @@ import { output } from '../../utils/output';
 const VIEW_LOGS_MESSAGE = `Hint: Try "nx view-logs" to get structured, searchable errors logs in your browser.`;
 
 export function viewLogsFooterRows(failedTasks: number) {
-  if (failedTasks >= 2 && !isNxCloudUsed(readNxJson())) {
+  const [isCloudUsed, _] = isNxCloudUsed(readNxJson());
```

Comment:
The second parameter is not used so there is no need to define it.

```suggestion
  const [isCloudUsed] = isNxCloudUsed(readNxJson());
```